src/pps/pps.proto

```diff
@@ -313,6 +313,7 @@ message Worker {
 
 message Pipeline {
   string name = 1;
+  string project = 2;
```

Comment:
If this goes in as part of this PR, should it be a `Project` rather than a raw string?

Reply:
Yeah I agree :)